### PR TITLE
Update PHP-CS-Fixer to 3.0 and its config

### PR DESCRIPTION
**Pull request by sidz**
This PR updates PHP-CS-Fixer to 3.0 and it config file

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -118,7 +117,9 @@
         ],
         'logical_operators' => true,
         'native_constant_invocation' => true,
-        'native_function_invocation' => true,
+        'native_function_invocation' => [
+            'include' => ['@internal'],
```
Comment (sidz) [inline .php-cs-fixer.dist.php:121]: Keep old behavior as it has been changed. More details https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.0.0/UPGRADE-v3.md#changed-default-values-of-options

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -158,11 +160,6 @@
             'identical' => false,
             'less_and_greater' => false,
         ],
-        'global_namespace_import' => [
```
Comment (sidz) [inline .php-cs-fixer.dist.php:161]: duplicates line 105

File: .php-cs-fixer.dist.php [hunk 3]
```diff
@@ -138,9 +139,10 @@
         ],
         'php_unit_set_up_tear_down_visibility' => true,
         'php_unit_strict' => true,
-        'php_unit_ordered_covers' => true,
+        'phpdoc_order_by_value' => [
+            'annotations' => ['covers'],
+        ],
         'php_unit_test_annotation' => [
-            'case' => 'snake',
```
Comment (sidz) [inline .php-cs-fixer.dist.php:143]: removed in 3.0

File: Makefile [hunk 4]
```diff
@@ -66,14 +66,14 @@ check_trailing_whitespaces:
 .PHONY: cs
 cs:	  	 	## Runs PHP-CS-Fixer
 cs: $(PHP_CS_FIXER)
-	$(PHP_CS_FIXER) fix -v --cache-file=$(PHP_CS_FIXER_CACHE) --diff --diff-format=udiff
```
Comment (sidz) [inline Makefile:69]: `--diff-format` if removed. and `udiff` is default diff format in 3.0

**Review by maks-rafalko — APPROVED** [review]

**Pull request merged**